src/main/java/uk/gov/hmcts/darts/arm/component/impl/ArmRetentionEventDateCalculatorImpl.java

```diff
@@ -47,11 +47,19 @@ public boolean calculateRetentionEventDate(Integer externalObjectDirectoryId) {
                     return true;
                 } else if (ObjectRecordStatusEnum.STORED.getId() == externalObjectDirectory.getStatusId()) {
                     log.info("Updating retention date for ARM EOD {} ", externalObjectDirectoryId);
-                    UpdateMetadataResponse updateMetadataResponse = armDataManagementApi.updateMetadata(
-                        externalObjectDirectory.getExternalRecordId(), armRetentionDate);
-                    if (updateMetadataResponse.isError()) {
+                    UpdateMetadataResponse updateMetadataResponseMedia = armDataManagementApi.updateMetadata(
```

Comment:
Should these be separate calls? 
Should we be linking the ids in some way? The arm payloads do not distinguish the nature of the confidence scores i.e. media, case document etc

Reply:
It should be one single call to updateMetadata. Check how RetentionDate is passed to updatemetadata. Its dynamically fetched in ArmRetentionEventDateCalculatorImpl.getDocumentRetentionDate.